### PR TITLE
Assingment 2b.

### PR DESCRIPTION
**Pull request by JustAntoRS**
Checked with google tool

### CODE REVIEW

File: Assignment2/JustAntoRS-fromJSONtoJSON-LD.jsonld [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+    "@context":{
+        "xsd":"http://www.w3.org/2001/XMLSchema#",
+        "prop": "http://www.examples.org/properties"
```
Comment (paoespinozarias) [inline Assignment2/JustAntoRS-fromJSONtoJSON-LD.jsonld:4]: Add / or # at the end of URI definition

File: Assignment2/JustAntoRS-fromRDFtoJSON-LD.jsonld [hunk 2]
```diff
@@ -0,0 +1,49 @@
+{
+    "@context":{
+        "xsd": "http://www.w3.org/2001/XMLSchema#",
+        "prop": "http://www.examples.org/properties"
```
Comment (paoespinozarias) [inline Assignment2/JustAntoRS-fromRDFtoJSON-LD.jsonld:4]: Add / or # at the end of URI definition

**Review by paoespinozarias — CHANGES_REQUESTED** [review]

File: Assignment2/JustAntoRS-fromJSONtoJSON-LD.jsonld (hunk 1 above)
Comment (JustAntoRS) [reply Assignment2/JustAntoRS-fromJSONtoJSON-LD.jsonld]: Thanks for spotting out the typo, fixed it :)

File: Assignment2/JustAntoRS-fromRDFtoJSON-LD.jsonld (hunk 2 above)
Comment (JustAntoRS) [reply Assignment2/JustAntoRS-fromRDFtoJSON-LD.jsonld]: Thanks for spotting out the typo, fixed it :)

**Review by paoespinozarias — APPROVED** [review]
ok!

**Pull request merged**